tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(TPLGS
 "cavs-gain-hdmi\;cavs-gain-hdmi"
 # CAVS HDA topology with gain-based pipelines
 "cavs-gain-hdmi\;cavs-gain-hda\;HDA_CONFIG=gain"
-# CAVS HDA topology with mixer-based pipelines
+# CAVS HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
```

Comment:
while we're at it, can we try to cleanup the HDMI stuff, and make it a subset that's included in the generic case?
We should also rename those files so that they match what the kernel uses, otherwise we have a renaming that will get in the way of deployment/testing.